com.unity.render-pipelines.core/Editor/RenderPipeline/RenderPipelineGlobalSettingsUI.Drawers.cs

```diff
@@ -22,7 +22,8 @@ public static void DrawShaderStrippingSettings(ISerializedRenderPipelineGlobalSe
             using (new EditorGUI.IndentLevelScope())
             {
                 EditorGUILayout.PropertyField(serialized.shaderVariantLogLevel, Styles.shaderVariantLogLevelLabel);
-                EditorGUILayout.PropertyField(serialized.exportShaderVariants, Styles.exportShaderVariantsLabel);
+                if (Unsupported.IsDeveloperMode())
+                    EditorGUILayout.PropertyField(serialized.exportShaderVariants, Styles.exportShaderVariantsLabel);
```

Comment:
I think internally checks for the EditorPrefBool("DeveloperMode") that is enabled when you type `internal` into the about. I think any user will be able to get it. Just in case, but for now they are not using it as this is only for HDRP devs, and now URP devs.

Reply:
I don't think it's meant for Unity users to know about being able to type "internal" into the about dialog, right?

My understanding is that with this change, what users will see is the "Export Shader Variants" dialog option disappearing from the Global Settings. I'd like to make sure this change is explained. Also you are now referring to this dialog option in the public changelogs, which can potentially create confusion if they are not able to see the option.

Reply:
Removed